src/Pages/Ayuda.tsx

```diff
@@ -119,6 +123,40 @@ const Ayuda: React.FC = () => {
             </div>
           </div>
         </div>
+        <div className="flex w-3/5 mt-16 mx-auto  justify-center ">
+          <div className=" ml-5">
+            <p className="flex text-center mb-7 text-2xl font-bold">
+              Lo mas buscado
+            </p>
+            <ul className="flex flex-col w-85 text-sm font-semibold underline text-custom-blue-ligth cursor-pointer">
+              <li className="border-b-2 pt-5 pb-3 hover:text-custom-yellow hover:underline">
+                ¿Cuáles son los medios de contacto oficiales de Coppel?
+              </li>
+              <li className="border-b-2 pt-5 pb-3 hover:text-custom-yellow hover:underline">
+                ¿Cómo puedo hacer valida mi garantía de motos?
+              </li>
+              <li className="border-b-2 pt-5 pb-3 hover:text-custom-yellow hover:underline">
+                ¿En cuanto tiempo llega mi pedido de Marketplace?
+              </li>
+              <li className="border-b-2 pt-5 pb-3 hover:text-custom-yellow hover:underline">
+                ¿Cómo sé si un producto es de un vendedor externo o de Coppel?
+              </li>
+              <li className="border-b-2 pt-5 pb-3 hover:text-custom-yellow hover:underline">
+                ¿Qué es Coppel Max?
+              </li>
+            </ul>
+          </div>
+          <div className="w-96 ml-5">
+            <img
+              className="w-full h-full"
+              src="https://centrodeayuda.coppel.com/file-asset/Girl_in_jacket?v=1"
+              alt="centroAyuda"
```

Comment:
No es buena practica ponerle camel case al `alt`, porque el `alt` sirve para cuando al imagen no carga se muestre el texto del `alt`

Aparte la imagen es de una persona viendo el telefono

```suggestion
              alt="Persona texteando con un telefono"
```